### PR TITLE
Add Oney RoundEnd sound

### PR DESCRIPTION
**Pull request by Colin-Tel**
## About the PR

"Yeah you ever heard of consequences in video games? Maybe don't FUCKING die next time."

## Why / Balance

Adei agreed it'd be a good add.

**Changelog**

no

### CODE REVIEW

**Comment by ghost** [comment]
I can attest, so did the other two project managers at this time (BasedPugilist, Kurzaen.)

File: Resources/Audio/Announcements/RoundEnd/oney.ogg
Comment (Axionyxx) [inline Resources/Audio/Announcements/RoundEnd/oney.ogg:1]: This file is stereo, shouldn't they be mono?

**Review by Axionyxx — DISMISSED** [review]

File: Resources/Audio/Announcements/RoundEnd/oney.ogg
Comment (Axionyxx) [reply Resources/Audio/Announcements/RoundEnd/oney.ogg:1]: Everything else in that folder is stereo too it seems

**Comment by therealDLondon** [comment]
Headmin stamp of approv

File: Resources/Audio/Announcements/RoundEnd/oney.ogg
Comment (deltanedas) [reply Resources/Audio/Announcements/RoundEnd/oney.ogg:1]: (only sounds played ingame need to be mono, engine makes them stereo based on position and stuff)

**Comment by Colin-Tel** [comment]
🥺

**Pull request merged**